src/main/resources/templates/animal/about.ftl

```diff
@@ -0,0 +1,1226 @@
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</footer>
+</div>
+</div>
+<div class="qode-demos-toolbar" data-demo="bridge82.qodeinteractive.com">
+
+
+    <noscript>
+        <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-WCC77WV"
+                height="0" width="0" style="display:none;visibility:hidden"></iframe>
+    </noscript>
+    <script type="text/javascript">
+        /* <![CDATA[ */
+        var wpcf7 = {
+            "apiSettings": {
+                "root": "https:\/\/bridge82.qodeinteractive.com\/wp-json\/contact-form-7\/v1",
+                "namespace": "contact-form-7\/v1"
+            }, "cached": "1"
+        };
+        /* ]]> */
+
+    </script>
+    <script src="https://bridge82.qodeinteractive.com/wp-content/cache/minify/0fef6.js"></script>
+    <script type="text/javascript"
+            src="https://toolbar.qodeinteractive.com/_qode_toolbar/assets/js/jquery.lazy.min.js?ver=5.3"></script>
+    <script type="text/javascript"
+            src="https://toolbar.qodeinteractive.com/_qode_toolbar/assets/js/qode-demos-modules.js?ver=5.3"></script>
+    <script type="text/javascript"
+            src="https://toolbar.qodeinteractive.com/_qode_toolbar/assets/js/swiper.min.js?ver=5.3"></script>
+    <script type="text/javascript">
+        /* <![CDATA[ */
+        var qodeLike = {"ajaxurl": "https:\/\/bridge82.qodeinteractive.com\/wp-admin\/admin-ajax.php"};
+        /* ]]> */
+
+    </script>
+    <script src="https://bridge82.qodeinteractive.com/wp-content/cache/minify/3f0fa.js"></script>
+    <script type="text/javascript">
+        /* <![CDATA[ */
+        var uiAutocompleteL10n = {
+            "noResults": "No results found.",
+            "oneResult": "1 result found. Use up and down arrow keys to navigate.",
+            "manyResults": "%d results found. Use up and down arrow keys to navigate.",
+            "itemSelected": "Item selected."
+        };
+        /* ]]> */
+
+    </script>
+    <script src="https://bridge82.qodeinteractive.com/wp-content/cache/minify/5f16d.js"></script>
+    <script type="text/javascript">
+        jQuery(document).ready(function (jQuery) {
+            jQuery.datepicker.setDefaults({
+                "closeText": "Close",
+                "currentText": "Today",
+                "monthNames": ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"],
+                "monthNamesShort": ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"],
+                "nextText": "Next",
+                "prevText": "Previous",
+                "dayNames": ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"],
+                "dayNamesShort": ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"],
+                "dayNamesMin": ["S", "M", "T", "W", "T", "F", "S"],
+                "dateFormat": "MM d, yy",
```

Comment:
What is the purpose of this part of the code?

Reply:
Deleted

Reply:
Deleted